bin/next-build-tools.js

```diff
@@ -136,6 +137,19 @@ program
 		});
 
 	program
+		.command('deploy-vcl [folder]')
+		.description('Deploys VCL in [folder] to the fastly service given in the FASTLY_SERVICE_ID env var.  Also requires FASTLY_KEY env var')
+		.option('-m, --main <main', 'Set the name of the main vcl file (the entry point).  Defaults to "main.vcl"')
```

Comment:
why not just specify the path to the VCL file directly?  and then default to wherever you think is best to put the main VCL file?